elastalert/ruletypes.py

```diff
@@ -160,26 +160,39 @@ class ChangeRule(CompareRule):
 
     def compare(self, event):
         key = hashable(lookup_es_key(event, self.rules['query_key']))
-        val = lookup_es_key(event, self.rules['compare_key'])
-        if not isinstance(val, bool) and not val and self.rules['ignore_null']:
-            return False
-        changed = False
+        values = []
+        elastalert_logger.info(" Inside Change Rule " + str(self.occurrences))
+        for val in self.rules['compare_key'].split(","):
+            lookup_value = lookup_es_key(event, val)
+            values.append(lookup_value)
+        elastalert_logger.info("  " + str(values) + " " + str(values))
+        for val in values:
+            if not isinstance(val, bool) and not val and self.rules['ignore_null']:
```

Comment:
Yes I agree with you . Currently our need is to apply it to any fields ,thats why I did that way . I'll take it as separate feature (where user should also be able to provide threshold change in values ,in order to get alert) 
Please let me know if its ok.